tests/conftest.py

```diff
@@ -136,3 +141,38 @@ def reload_directory_structure(tmp_path_factory: pytest.TempPathFactory):
 @pytest.fixture(scope="function")
 def logging_config() -> dict:
     return deepcopy(LOGGING_CONFIG)
+
+
+@pytest.fixture
+def short_socket_name(tmp_path, tmp_path_factory):  # pragma: py-win32
+    max_sock_len = 100
```

Comment:
I'm usually against comments, but a comment explaining the reason for the number 100 would be cool.